core/storage/learner_group/gae_models_test.py

```diff
@@ -261,3 +261,29 @@ def test_get_by_facilitator_id(self) -> None:
             learner_group_models.LearnerGroupModel.get_by_facilitator_id(
                 'user_1'))
         self.assertEqual(learner_group_model[0].id, 'learner_group_32')
+
+    def test_get_by_invited_learner_user_id(self) -> None:
+        """Test get_by_invited_learner_user_id."""
+        learner_grp_models = (
+            learner_group_models.LearnerGroupModel
+                .get_by_invited_learner_user_id(
+                    'user_1'))
```

Comment:
It would be better to name the users (at time of creation in setUp()) something like 'user_invited_to_group_5_but_not_group_3' or similar, so that the test is more clearly true and readers don't need to keep track of "who is user_5 and how are they different from user_1".

Reply:
updated!